src/CMakeLists.txt

```diff
@@ -73,6 +73,15 @@ target_link_libraries(${PROJECT_NAME}
     ${PORTAUDIO_LIBRARIES}
 )
 
+if(NOT Gnuradio_VERSION VERSION_LESS "3.8")
+    target_link_libraries(${PROJECT_NAME}
+        gnuradio::gnuradio-analog
+        gnuradio::gnuradio-blocks
+        gnuradio::gnuradio-digital
+        gnuradio::gnuradio-filter
+    )
+endif()
```

Comment:
For 3.7 the linked libraries are defined in GNURADIO_ALL_LIBRARIES, which should be empty for 3.8.